addons/test_mail/tests/test_mail_message.py

```diff
@@ -57,7 +57,6 @@ def test_empty_message(self):
         self.assertEqual(len(message.attachment_ids), 1)
         self.assertFalse(is_html_empty(message.body))
         self.assertEqual(len(message.sudo().notification_ids), 1)
-        self.assertEqual(message.notified_partner_ids, self.partner_admin)
```

Comment:
Isn't it true anymore ? Since we keep notifications and stuff like that ? same question for other removal of notified_partner_ids .

Reply:
I think it was just used to check before/after "delete" (as we cleaned it), but now the value is the same before and after so it's not really useful to keep checking it if we're not expecting it to change.